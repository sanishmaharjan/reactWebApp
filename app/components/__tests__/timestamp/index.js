import React from 'react';
import renderer from 'react-test-renderer';
import TimeStamp from '../../timeStamp';

describe('TimeStamp', () => {
    it('render correctly', () => {
        const date = new Date('Sun Dec 03 2017 19:48:55 GMT+0545 (+0545)');
        const time = date.toLocaleTimeString([], {hour: '2-digit', minute: '2-digit'});
        const timestamp = renderer.create(<TimeStamp time={time}/>).toJSON();

        expect(timestamp.children.length).toBe(1);
        expect(timestamp).toMatchSnapshot();
    });
});