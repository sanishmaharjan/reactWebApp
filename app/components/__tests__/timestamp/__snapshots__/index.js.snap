// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TimeStamp render correctly 1`] = `
<span>
  19:48
</span>
`;
